purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -230,6 +230,10 @@ internal class PurchasesOrchestrator(
         }
     }
 
+    fun getLatestConfiguration(): PurchasesConfiguration {
+        return configuration.copy(appUserID = this.appUserID)
```

Comment:
We could also make `configuration` always be the latest? Otherwise we might get confused between `configuration` and `getLatestConfiguration()`, and when to use which.

Reply:
yeah good point... can probably clean this up a bit.